src/shogun/neuralnets/NeuralNetwork.h

```diff
@@ -442,6 +442,18 @@ friend class CDeepBeliefNetwork;
 	 * default value is -1
 	 */
 	float64_t gd_error_damping_coeff;
+
+	/** Used in RmsProp stochastic optimization 
```

Comment:
I **think** you need to use [this syntax](https://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup) to group the documentation of more than one member together.